infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/OperationalPointDao.kt

```diff
@@ -125,7 +131,7 @@ class OperationalPointDao(
                 "y_min" to searchBox?.bbox?.y?.min,
                 "y_max" to searchBox?.bbox?.y?.max,
                 "layout_srid" to LAYOUT_SRID.code,
-            )
+            ) + if (ids != null) mapOf("ids" to ids.map { it.intValue }.toTypedArray()) else mapOf()
```

Comment:
Eikös `ids`:in voisi vaan välittää tuolle joka tapauksessa? `$idsFragment`:han vähän niinkuin hoitaa jo olemassaolotarkastelun, jolloin `ids`:in voisi vaan välittää kutsulle tapauksessa kuin tapauksessa ja sitten sitä joko käytetään tai ei